src/transformers/models/bert/modeling_bert.py

```diff
@@ -197,6 +206,8 @@ def forward(
         if token_type_ids is None:
             token_type_ids = torch.zeros(input_shape, dtype=torch.long, device=self.position_ids.device)
 
+        token_type_ids = self.token_type_ids[:, :seq_length]
+
```

Comment:
This won't work if PyTorch is < 1.6.0, as `self.token_type_ids` would be undefined. I would put it behind a `if hasattr(self, "token_type_ids")`.

Reply:
@LysandreJik thanks for reviewing the changes,I have added the version check. As discussed off-line, I would appreciate if you could cross check if token_type_ids are not None when loading the traced model using the script above, as per my test pushing it to the first if block failed with the original error.

 if token_type_ids is None:
            if hasattr(self, "token_type_ids"):
                token_type_ids = self.token_type_ids[:, :seq_length]
            else:
                token_type_ids = torch.zeros(input_shape, dtype=torch.long, device=self.position_ids.device)